tigerpath/majors_and_certificates/majors/CBE.json

```diff
@@ -14,6 +14,11 @@
             "type": "Departmental Representative",
             "name": "A. James Link",
             "email": "ajlink@princeton.edu"
+        },
+        {
```

Comment:
https://ua.princeton.edu/academic-units/department-chemical-and-biological-engineering